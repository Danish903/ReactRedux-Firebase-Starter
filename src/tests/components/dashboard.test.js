import React from "react";
import {shallow} from "enzyme";
import Dashboard from "../../components/dashboard";

test("should set up test from dashboard correctly", () => {
  const wrapper = shallow(<Dashboard />);
  expect(wrapper).toMatchSnapshot();
});
