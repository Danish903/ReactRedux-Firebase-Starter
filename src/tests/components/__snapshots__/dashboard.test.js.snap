// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should set up test from dashboard correctly 1`] = `
<div>
  <h1>
    dashboard
  </h1>
</div>
`;
